service/src/main/resources/db/migration/V1.57__Update_Beacon_Search_View_Legacy_Created_Date.sql

```diff
@@ -0,0 +1,65 @@
+DROP VIEW beacon_search;
+CREATE OR REPLACE VIEW beacon_search AS
+SELECT
+    id,
+    CAST(data -> 'beacon' ->> 'firstRegistrationDate' AS TIMESTAMPTZ) AS created_date,
+    last_modified_date,
+    COALESCE(c.beacon_status, l.beacon_status) AS beacon_status,
+    hex_id,
+    owner_name,
+    owner_email,
+    NULL as account_holder_id,
+    NULL as account_holder_name,
+    NULL as account_holder_email,
+    use_activities,
+    'LEGACY_BEACON' AS beacon_type,
+    data -> 'beacon' ->> 'manufacturerSerialNumber' AS manufacturer_serial_number,
+    data -> 'beacon' ->> 'cospasSarsatNumber' AS cospas_sarsat_number
+FROM
+    legacy_beacon l
+        LEFT JOIN
+    (
+        SELECT
+            legacy_beacon_id,
+            'DELETED' as beacon_status
+        FROM
+            legacy_beacon_claim_event
+    )
+        c
+    ON c.legacy_beacon_id = l.id
+UNION ALL
+SELECT
+    b.id,
+    b.created_date,
+    b.last_modified_date,
+    b.beacon_status,
+    b.hex_id,
+    o.full_name as owner_name,
+    o.email as owner_email,
+    a.id as account_holder_id,
+    a.full_name as account_holder_name,
+    a.email as account_holder_email,
+    u.use_activities,
+    'BEACON' AS beacon_type,
+    manufacturer_serial_number,
+    NULL as cospas_sarsat_number
+FROM
+    beacon b
+        LEFT JOIN
+    beacon_owner o
+    ON b.id = o.beacon_id
+        LEFT JOIN
+    account_holder a
+    ON a.id = b.account_holder_id
+        LEFT JOIN
```

Comment:
LEFT_JOIN returns all the rows from the left table so just checking do you want all the rows from beacon and account_holder?

Reply:
Yeah I didn't want to mess with this too much. This was already existing, and is used by other searches. I changed where the legacy created_date is pulled from and also added a couple of extra account holder fields. 

Probably not the best query but it's not broke so not looking to fix it right now